dbt/models/iasworld/schema/iasworld.land.yml

```diff
@@ -24,7 +24,7 @@ sources:
                   additional_select_columns:
                     - taxyr
                     - parid
-                    - card
+                    - lline
```

Comment:
Whoops, turns out that `card` is a vestigal attribute on `land`, and we should be using `lline` as part of the composite primary key instead.

Reply:
We definitely should, as it's already correctly defined as the key in [the docs](https://ccao-data.github.io/data-architecture/#!/source/source.athena.iasworld.land). Must've missed this during a prior review!